src/components/app-content.tsx

```diff
@@ -13,7 +14,9 @@ export const AppContentComponent: React.FC<IProps> = (props) => {
   return (
     // &dataflow with no assignment is returned as null
     urlParams.dataflow !== undefined
-      ? <DataflowAppContentComponent {...props} />
+      ? <Provider stores={createStores()}>
```

Comment:
Overwriting the provided `stores` seems to work here, but I'm not sure if there's a safer option?